### PR TITLE
Add function to get cost info of specific account under an organization

### LINKED ISSUE #7: Add functionality to specify accounts under the org
- Setting account aliases itself incurs a certain decision-making cost. We want to be able to retrieve cost information for specific accounts under the organization even if the account alias is not set.

### PR DESCRIPTION
**Pull request by Rindrics**
close #7

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -34,9 +34,11 @@ $ aws-cost --help
 
     -k, --access-key [key]         AWS access key
     -s, --secret-key [key]         AWS secret key
+    -t, --session-token [key]      AWS session token
     -r, --region [region]          AWS region (default: us-east-1)
     -a, --role-arn [arn]           AWS role ARN to assume
 
+    --target-account [id]          Account ID to see cost
```
Comment (bvlion) [inline readme.md]: Troublesome, but no good idea comes to mind 🤔

File: src/account.ts [hunk 2]
```diff
@@ -1,14 +1,34 @@
...
 import AWS from 'aws-sdk';
 import { AWSConfig } from './config';
 import { showSpinner } from './logger';
 
 export async function getAccountAlias(awsConfig: AWSConfig): Promise<string> {
   showSpinner('Getting account alias');
 
+  if (awsConfig.targetAccount) {
+    const organizations = new OrganizationsClient({
+      region: 'us-east-1', // Organizations API is only available in us-east-1
```
Comment (bvlion) [inline src/account.ts:14]: Oh, I didn't know that !!

**Review by bvlion — APPROVED** [review]
I haven't run it, and I have added comments, but the content seems fine to me 🙆‍♂️

File: readme.md (hunk 1 above)
Comment (rindrics) [reply readme.md]: I added https://github.com/route06/aws-cost-cli/pull/20/commits/0eb8cb76888fef0ee10b106a2f818a076ad397a1

**Review by bvlion — APPROVED** [review]
LGTM 👍

**Comment by rindrics** [comment]
thx!

**Pull request merged**